packages/matchbox/src/components/Panel/Section.js

```diff
@@ -22,6 +22,7 @@ const Section = React.forwardRef(function Section(props, userRef) {
   const actions = getChild('Panel.Action', children);
   const content = excludeChild(['Panel.Action'], children);
   const paddingContext = React.useContext(PanelPaddingContext);
+  const appearanceContext = React.useContext(PanelAppearanceContext);
```

Comment:
Context is nice, but what happens if the user wants one Section normal, and another one inverted in the same Panel?

Reply:
Should we even support this use case though?

Reply:
Would be easy enough to pass through to specific sections. Is our use case for these appearance changes a dark mode or something else, because if it's a dark mode I could see it being annoying having to specify an inverted appearance on all panel sections

Reply:
I think this is the main use case for now. 
<img width="1280" alt="Screen Shot 2021-02-24 at 5 19 28 PM" src="https://user-images.githubusercontent.com/3903325/109073512-7abff480-76c4-11eb-82f1-d9bf6a35951b.png">

With dark mode enabled, I imagine "inverted" would probably stay dark, although we should talk about that when the time comes.